webapp/lib/view.dart

```diff
@@ -211,26 +212,42 @@ class ConfigurationViewTagListPartial {
       ..classes.add('configure-package__tags-list');
   }
 
-  void renderTagList(List<String> tags) {
+  void renderTagList(Map<String, bool> tags) {
     tagListElement.children.clear();
-    tags.forEach((tag) {
+    tags.forEach((tag, state) {
       var tagItem = new Element.li()
         ..classes.add('configure-package__tag-item')
         ..text = tag;
       tagItem.onClick.listen((event) {
         var selectedTag = (event.target as Element);
         controller.command(controller.UIAction.configurationTagSelected, new controller.ConfigurationTagData(selectedTag: selectedTag.text.trim()));
-        tagListElement.children.forEach((t) => t.classes.remove('configure-package__tag-item-active'));
-        selectedTag.classes.add('configure-package__tag-item-active');
+      });
+      tagItem.onDragEnter.listen((event) => event.preventDefault());
+      tagItem.onDragOver.listen((event) {
+        event.preventDefault();
+        (event.target as Element).classes.add('configure-package__tag-item-active');
+      });
+      tagItem.onDragLeave.listen((event) => (event.target as Element).classes.remove('configure-package__tag-item-active'));
+      tagItem.onDrop.listen((event) {
+        event.preventDefault();
+        var dropTarget = (event.target as Element);
+        dropTarget.classes.remove('configure-package__tag-item-active');
+        if (dropTarget.classes.contains('configure-package__tag-item')) {
+          var responseData = jsonDecode(event.dataTransfer.getData("Text"));
+          responseData.forEach((language, text) {
+            controller.command(controller.UIAction.addConfigurationResponseEntries,
+              new controller.ConfigurationResponseData(parentTag: dropTarget.text, language: language, text: text));
```

Comment:
As discussed earlier - this should be done in a row at once, rather than one text message at a time - nothing to do now about it, but to be picked up in the future PR that adjusts the model

Reply:
ok